### ISSUE TITLE
0995 - 508 audit  defect 5 WCAG 1.3.1 fake headings in the evidence summary page

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#109038]
## Issue
- This ticket is for tracking fixes for Defect 5 from the recent 508 audit on the 0995

> WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.
> 
> On the File A Supplemental Claim | Veterans Affairs screen (Step3 of 5New and relevant evidence), under We’re requesting records from these VA locations:, text, such as the names of the VA locations, are not read as headings. (New: 04/04/2025)
> 
> On the File A Supplemental Claim | Veterans Affairs screen (Step5 of 5Review application), the visual presentation of information cannot be understood by assistive technology as it is read as unstructured text. (New: 04/04/2025) - FIXED

Step 5 of 5 has been confirmed fixed by ADE. Step 3 of 5 needs to be fixed. From checking out the code it seems that no platform code is involved. The problem is that the text looks like a heading, but is coded as a div. Need to change all of these to headings.

<img width="528" alt="Image" src="https://github.com/user-attachments/assets/6ffce885-8a52-494d-8544-2c347c8baca4" />

## Acceptance Criteria
- The bolded text under the evidence summary page will be h5s and not just divs of text

## Resources
[[ADE 508 Audit]: 508 Audit Report for 43785 va.gov Forms](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106900#top)
[[ADE 508 Audit]: 508 Audit Report for 43785 va.gov Forms - Defect 5 | Critical #107040](https://github.com/department-of-veterans-affairs/va.gov-team/issues/107040)
